tensorflow/python/keras/utils/dataset_creator.py

```diff
@@ -79,6 +80,43 @@ def dataset_fn(input_context):
       with distributed training. See `tf.distribute.InputOptions` for more
       information.
   """
+  class Builder(tfds.core.GeneratorBasedBuilder):
+  """DatasetBuilder for my_dataset dataset."""
```

Comment:
This is now a class within a class and also it is empty. It is actually a python syntax error.

Please make sure the code you write passes tests before sending a PR